.pre-commit-config.yaml

```diff
@@ -43,6 +43,7 @@ repos:
     rev: 20.8b1
     hooks:
       - id: black
+        exclude: ^upgrade_analysis/odoo_patch/
```

Comment:
As I maintain the template, I'm interested on knowing... what's the problem with pre-commit in these files?

Reply:
odoo_patch is a place where odoo code is patched. to simplify diff analysis. 
some times, we require overwritting odoo function.
in that case, we don't want to apply black. (and all other checks) because it is a copy of odoo code.

Reply:
It is actually the case that so far we have been able to copy large swaths of code, so for now maybe we can do without this exclude? We can always add individual patches if needed.

Reply:
Hi @Yajo : I remove pre-commit exception, and so fixed some style errors.

@StefanRijnhart : I received 3 B902 errors I decided to ignore. what do you think ? it's regarding patches part.

here is the diff : 
https://github.com/OCA/server-tools/pull/1941/commits/d3d94f1d8f79a256ebbe736bde754dfecf82bea5#diff-c5d89b073f972f8dc85c7bb7e8aff14db2b2499783fcc3763fd234062de73393R1

thanks.

Reply:
Yes, very good. Thanks!